incrementals-publisher/test/permissions-test.js

```diff
@@ -25,13 +25,11 @@ describe('The Permissions helpers', function() {
         path.resolve('./test/fixtures-bad-scm-url-archive.zip'),
         entries,
         perms,
-        folderMetadataParsed.owner,
-        folderMetadataParsed.repo,
         buildMetadataParsed.hash
       ),
       {
         name: 'Error',
-        message: 'ZIP error: Error: Wrong URL in /project/scm/url'
+        message: 'ZIP error: Error: Missing <url> section in <scm> of io/jenkins/tools/bom/bom/2.176.1-rc41.149af85f094d/bom-2.176.1-rc41.149af85f094d.pom'
```

Comment:
Is this actually correct? Should it have that specific version of the bom in this message?

Reply:
Oh, this is a test, I see. I was misreading.

Reply:
The assertion here previously was verifying a condition that as of this PR is not considered incorrect, so I had to amend the test ZIP to delete just the `<url>` so as to still verify that _some_ checks are being done.